src/System.Runtime/src/System/Reflection/AssemblyNameProxy.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*============================================================
+**
+**
+**
+**
+**
+**
+** Purpose: Remotable version the AssemblyName
+**
+**
+===========================================================*/
+namespace System.Reflection {
+    using System;
+    using System.Runtime.Versioning;
+
+    [System.Runtime.InteropServices.ComVisible(true)]
+    public class AssemblyNameProxy : MarshalByRefObject
```

Comment:
@weshaggard It should. However, it calls `AssemblyName.GetAssemblyName(string)` which is not available at the moment. I did not even add the method here so that we can find it easily in the diff together with all the other still-missing APIs being tracked [here](https://github.com/dotnet/corefx/issues/10530)